src/views/User/GameHistoryTable.tsx

```diff
@@ -393,6 +396,11 @@ export function GameHistoryTable(props: GameHistoryProps) {
                                     "board_size" + (X && X.annulled ? " annulled" : ""),
                                 render: (X) => `${X.width}x${X.height}`,
                             },
+                            {
+                                header: _("Hd"),
```

Comment:
Can we change `Hd` to `HC` - that's the common abbreviation that I've seen.

Also, let's provide some context to our translators, instead of `_("HC")` do `pgettext("Handicap abbreviation", "HC")` that way translators can change that to whatever language appropriate abbreviation is used.